docs/pages/server-access/getting-started.mdx

```diff
@@ -0,0 +1,479 @@
+   </Admonition>
+
+4. You should now be able to view both Nodes in the Teleport Web interface after logging in as `tele-admin`:
+
+   <Figure
+     align="center"
+     bordered
+     caption="Both Nodes in the Web UI"
+   >
+     ![Both Nodes in the Web UI](../../img/server-access/teleport_ui.png)
+   </Figure>
+
+## Step 3/5. SSH into the server
+
+Now, that we've got our cluster up and running, let's see how easy it is to connect to our two Nodes. 
+
+We can use `tsh` to SSH into the cluster:
+
+1. On your local machine, log in through `tsh`: 
+
+   ```bash
+   # Log in through tsh
+   tsh login --proxy=tele.example.com:443 --user=tele-admin
+   ```
+   
+   You'll be prompted to supply the password and One Time Passcode we set up previously.
+
+2. `tele-admin` will now see:
+
+   ```bash
+   Profile URL:        https://tele.example.com:443
+     Logged in as:       tele-admin
+     Cluster:            tele.example.com
+     Roles:              access, editor
+     Logins:             root, ubuntu, ec2-user
+     Kubernetes:         disabled
+     Valid until:        2021-04-30 06:39:13 -0500 CDT [valid for 12h0m0s]
+     Extensions:         permit-agent-forwarding, permit-port-forwarding, permit-pty
+   ```
+
+   `tele-admin` is now logged into the `tele.example.com` cluster and *Bastion Host* Node through Teleport SSH.
+
+3. `tele-admin` can now execute the following to find the cluster's `nodenames`. `nodenames` are used for establishing SSH connections:
+
+   ```bash
+   # Display cluster resources
+   tsh ls
+   ```
+
+   The *Bastion Host* Node is located on the bottom line below:
+
+   ```bash                        
+   Node Name        Address        Labels                                 
+   ---------------- -------------- -------------------------------------- 
+   ip-172-31-35-170 ⟵ Tunnel                                              
+   ip-172-31-41-144 127.0.0.1:3022 env=example, hostname=ip-172-31-41-144 
+   ```
+
+4. `tele-admin` can SSH into the *Bastion Host* Node by running the following command locally:
+
+   ```bash
+   # Use tsh to ssh into a Node
+   tsh ssh root@ip-172-31-41-144
+   ```
+
+   Now, they can:
+
+   - Connect to other Nodes in the cluster by using the appropriate IP address in the `tsh ssh` command.  
+   - Traverse the Linux file system.  
+   - Execute desired commands.  
+
+   All commands executed by `tele-admin` are recorded and can be replayed in the Teleport Web interface.
+   
+   The `tsh ssh` command allows one to do anything they would if they were to SSH into a server using a third-party tool. Compare the two equivalent commands:
+
+<Tabs>
+  <TabItem label="tsh">
+    ```bash
+    tsh ssh root@ip-172-31-41-144
+    ```
+  </TabItem>
+  <TabItem label="ssh">
+    ```bash
+    ssh -J tele.example.com root@ip-172-31-41-144
+    ```
+  </TabItem>
+</Tabs>
+
+## Step 4/5. Use tsh and the universal resource catalog to introspect the cluster
```

Comment:
Let's remove universal.

Reply:
Just want to verify - I should remove universal? And, not use the full product name?

Reply:
I've corrected the typo here to `unified` since it still appears in the marketing guidelines. However, happy to alter this pending other branding changes.